tests/app/helpers.py

```diff
@@ -247,6 +247,25 @@ def setup_dummy_services_including_unpublished(self, n):
             )
             db.session.commit()
 
+    def setup_dos_2_framework(self, status='open', clarifications=True):
```

Comment:
No - I left a note in the PR message - also needs associated Lots to be set up which I couldn't figure out how to do with fixtures.